service/src/test/java/uk/gov/hmcts/reform/fpl/service/DirectionHelperServiceTest.java

```diff
@@ -48,7 +53,15 @@
 @ExtendWith(SpringExtension.class)
 class DirectionHelperServiceTest {
```

Comment:
1498 lines, I feel like this needs to be split up. Maybe have a package for it with an abstract class with the build/populate methods and common variables and then move each of the nested classes to a separate class which extends the abstract class

Reply:
yep, I wanna refactor DirectionHelperService to separate services